rules/gcs.bzl

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Skylark rule for pulling a file from GCS bucket.
+"""
+
+_GCS_FILE_BUILD = """
+package(default_visibility = ["//visibility:public"])
```

Comment:
I think this line should be the first line after the comments

Reply:
this is the content of the produced BUILD file, so needs to be inside this variable. The alternative (which might be good) is to have this in a template file, and use repository_ctx.template (https://docs.bazel.build/versions/master/skylark/lib/repository_ctx.html#template)

Reply:
I will keep content of BUILD file here, but can move it to a template if we want.